test/test_sbol2_sbol3_direct.py

```diff
@@ -149,5 +149,24 @@ def test_2to3_collection_conversion(self):
             self.assertFalse(file_diff(str(tmp2), str(TEST_FILES / 'sbol_3to2_collection.xml')))
     '''
 
+    def test_empty_componentDefinition(self):
+            """Test ability to convert a simple part from SBOL2 to SBOL3"""
+            # Load an SBOL2 document and check its contents
+            doc2 = sbol2.Document()
+            doc2.read(TEST_FILES / 'empty_componentDefinition.xml')
+            # Convert to SBOL3 and check contents
+            doc3 = convert2to3(doc2, use_native_converter=True)
+            self.assertEqual(len(doc3.validate()), 0)
+            with tempfile.TemporaryDirectory() as tmpdir:
+                tmp3 = Path(tmpdir) / 'doc3.nt'
+                doc3.write(tmp3)
+                #self.assertFalse(file_diff(str(tmp3), str(TEST_FILES / 'BBa_J23101_patched.nt')))
```

Comment:
This should be replaced with an SBOL3 serialization of the expected values for `empty_componentDefinition.nt`